src/Mjolnir.ts

```diff
@@ -399,13 +399,40 @@ export class Mjolnir {
         }
     }
 
+    public async getProtectionSettings(protectionName: string): Promise<{ [setting: string]: any }> {
+        const eventName = `org.matrix.mjolnir.set.${protectionName}`;
```

Comment:
Nit: `set` is a weird name. I assume it's a shortcut for "setting", but I believe that the latter would be easier to parse when someone ends up debugging the state events.